cubedash/summary/_stores.py

```diff
@@ -128,18 +128,19 @@ def refresh_product(
     ):
         our_product = self.get_product_summary(product.name)
 
+        _LOG.info(f"Refreshing if {our_product.last_refresh_age} is older than {refresh_older_than}")
```

Comment:
I'd prefer to add `age=` as a struct-logged parameter below -- I was trying to keep UI messages out of the inner lib, as it's callable in Python code directly.